modules/gitops-in-built-permissions.adoc

```diff
@@ -0,0 +1,18 @@
+// Module is included in the following assemblies:
+//
+// * openshift-docs/cicd/gitops/configuring-a-cluster-to-use-gitops.adoc
+
+[id="in-built-permissions_{context}"]
+= In-built permissions for Argo CD
+
+This section lists the permissions that are granted to ArgoCD to manage specific cluster-scoped resources which include platform operators, optional OLM operators, and user management. Note that ArgoCD is not granted `cluster-admin`.
```

Comment:
```suggestion
This section lists the permissions that are granted to ArgoCD to manage specific cluster-scoped resources, which include platform operators, optional OLM operators, and user management. Note that ArgoCD is not granted `cluster-admin`.
```

Reply:
Note that ArgoCD is not granted `cluster-admin` permissions.